tests/globalhelper/crd.go

```diff
@@ -0,0 +1,70 @@
+package globalhelper
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	crdRetryIntervalSecs = 5
+)
+
+func CreateAndWaitUntilCrdIsReady(crd *apiextv1.CustomResourceDefinition, timeout time.Duration) error {
+	_, err := APIClient.CustomResourceDefinitions().Create(
+		context.Background(),
+		crd,
+		metav1.CreateOptions{},
+	)
+	if err != nil {
+		return err
+	}
+
+	Eventually(func() bool {
+		runningCrd, err := APIClient.CustomResourceDefinitions().Get(
+			context.Background(),
+			crd.Name,
+			metav1.GetOptions{},
+		)
+		if err != nil {
+			glog.V(5).Info(fmt.Sprintf(
+				"crd %s is not ready, retry in 5 seconds", runningCrd.Name))
+
+			return false
+		}
+
+		for _, condition := range runningCrd.Status.Conditions {
+			if condition.Type == apiextv1.Established {
+				return true
+			}
+		}
+
+		return false
+	}, timeout, crdRetryIntervalSecs*time.Second).Should(Equal(true), "CRD is not ready")
```

Comment:
Might make sense to move the *time.Second up to the const definition since it's used twice in the same way

Reply:
Done.